src/components/FeatureViews/Courses/CourseClasses.js

```diff
@@ -268,234 +273,223 @@ const CourseClasses = () => {
-                            align="left"
-                            className={classes.alignTitleLeft}
-                        >
-                            Grade
-                        </Typography>
-                        <Typography
-                            variant="body1"
-                            align="left"
-                            className={classes.dataFontDate}
-                        >
-                            {gradeLvl(academicLevel)}
-                        </Typography>
-                    </Grid>
-                    <Grid item xs={2} md={4} lg={2} xl={2}>
-                        <Typography
-                            variant="body2"
-                            align="left"
-                            className={classes.alignTitleLeft}
+                <Grid item xs={6}>
+                    <AccessControlComponent
+                        permittedAccountTypes={[
+                            USER_TYPES.admin,
+                            USER_TYPES.receptionist,
+                            USER_TYPES.instructor,
+                        ]}
+                    >
+                        <IconButton
+                            className={classes.editcoursebutton}
+                            size="small"
+                            component={Link}
+                            to={`/registration/form/course_details/${id}`}
                         >
-                            Subject
-                        </Typography>
-                        <Typography
-                            variant="body1"
-                            align="left"
-                            className={classes.dataFontDate}
-                        >
-                            {name}
-                        </Typography>
-                    </Grid>
+                            <EditIcon />
+                        </IconButton>
+                    </AccessControlComponent>
+                </Grid>
+            </Grid>
+            <Grid container justify="flex-start" style={{ marginTop: '2.5em' }}>
+                <Grid item xs={2} md={4} lg={3} xl={2}>
+                    <Typography
+                        variant="body2"
+                        align="left"
+                        className={classes.alignTitleLeft}
+                    >
+                        Date
+                    </Typography>
+                    <Typography
+                        variant="body1"
+                        align="left"
+                        className={classes.dataFontDate}
+                    >
+                        {`${startingDate} - ${endingDate}`}
+                    </Typography>
+                </Grid>
+                <Grid item xs={2} md={4} lg={3} xl={2}>
+                    <Typography
+                        variant="body2"
+                        align="left"
+                        className={classes.alignTitleLeft}
+                    >
+                        Time
```

Comment:
shouldn't this be using the newest course availabilities component that @andrewjknapp made?